File: app/src/main/java/com/unis/gameplatfrom/model/GamesEntity.java
package com.unis.gameplatfrom.model;

import org.litepal.crud.LitePalSupport;

public class GamesEntity extends LitePalSupport {

     private int id;
     private int v;
     private String p;
     private String icon;
     private String packname;
     private String name;
     private String account;//判断用户的游戏
     private boolean isGame;//测试游戏是否存在
     private boolean isNewGame;//测试是否有新版本


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getV() {
        return v;
    }

    public void setV(int v) {
        this.v = v;
    }

    public String getP() {
        return p;
    }

    public void setP(String p) {
        this.p = p;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getPackname() {
        return packname;
    }

    public void setPackname(String packname) {
        this.packname = packname;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAccount() {
        return account;
    }

    public void setAccount(String account) {
        this.account = account;
    }

    public boolean isGame() {
        return isGame;
    }

    public void setGame(boolean game) {
        isGame = game;
    }

    public boolean isNewGame() {
        return isNewGame;
    }

    public void setNewGame(boolean newGame) {
        isNewGame = newGame;
    }
}

File: kotlin/src/main/java/com/unis/kotlin/entity/GamesEntity.kt
package com.unis.kotlin.entity

import org.litepal.crud.LitePalSupport

class GamesEntity : LitePalSupport() {

     var id: Int = 0
     var v: Int = 0
     var newVersion: Int = 0//记录上一个版本
     var gameId: Int = 0
     var p: String? = null
     var icon: String? = null
     var packname: String? = null
     var name: String? = null
     var account: String? = null//判断用户的游戏
     var isInstallGame: Boolean = false//测试游戏是否已安装
     var isDownGame: Boolean = false//是否开始下载
     var isLocal: Boolean = false//判断游戏是否存在本地
     var isNewGame: Boolean = false//测试是否有新版本

     var process: Int = 0//进度条


}